modules/social_features/social_topic/social_topic.module

```diff
@@ -34,11 +34,16 @@ function social_topic_preprocess_node(array &$variables) {
     $topic_type = $node->get('field_topic_type');
     $topic_type_entities = $topic_type->referencedEntities();
     if (count($topic_type_entities) === 1) {
-      foreach ($topic_type_entities as $topic) {
-        $variables['metadata'] = t('in @topic', ['@topic' => $topic->link()]);
+      $lang_id = \Drupal::languageManager()->getCurrentLanguage()->getId();
+      /** @var Drupal\taxonomy\Entity\Term $topic_type_entity */
+      foreach ($topic_type_entities as $topic_type_entity) {
+        if ($topic_type_entity->hasTranslation($lang_id)) {
```

Comment:
Is this really needed? My understanding was that Drupal automatically uses the current language's translation if it's available and uses the default if it's not.

If we need to do this for every entity/content type then this can create a lot of custom code when creating new entities/content types.

Reply:
Yes, it only applies for our custom taxonomy terms that have these custom preprocess functions (topic and event types). Bug report: OSSUPPORT-588